modules/ossm-security-cert-manage.adoc

```diff
@@ -44,96 +45,68 @@ spec:
+Use the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo example application] to verify that the workload certificates are signed by the certificates that were plugged into the CA. This requires you have `openssl` installed on your machine
 
-. Store the pod name in the variable `RATINGSPOD`.
-+
-[source,terminal]
-----
-$ RATINGSPOD=`oc get pods -l app=ratings -o jsonpath='{.items[0].metadata.name}'`
-----
+. Extracting certificates from bookinfo workloads
 +
-. Run the following commands to retrieve the certificates mounted on the proxy.
+Wait about 1 minute for the mTLS policty to take effect before retrieving the certificate chain of bookinfo. If Bookinfo was already deployed when you created the secret above, you need to restart the pods so they pick up the changes (see note above). Replace `<bookinfo-project>` with the project name where bookinfo workload is installed.
 +
-[source,terminal]
-----
-$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /var/run/secrets/istio/root-cert.pem > /tmp/pod-root-cert.pem
-----
-+
-The file `/tmp/pod-root-cert.pem` contains the root certificate propagated to the pod.
+[NOTE]
+====
+You must restart the bookinfo pods if they were already deployed. Use the following command to restart the pods: `oc -n <bookinfo-project> delete pods --all`.
+
+If the pods are deployed after the secret changes made above, then you can skip this task.
+====
 +
 [source,terminal]
 ----
-$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /etc/certs/cert-chain.pem > /tmp/pod-cert-chain.pem
+$ sleep 60
+$ oc -n <bookinfo-project> exec "$(oc -n <bookinfo-project> get pod -l app=productpage -o jsonpath={.items..metadata.name})" -c istio-proxy -- openssl s_client -showcerts -connect details:9080 > bookinfo-proxy-cert.txt
+$ sed -n '/-----BEGIN CERTIFICATE-----/{:start /-----END CERTIFICATE-----/!{N;b start};/.*/p}' bookinfo-proxy-cert.txt > certs.pem
+$ awk 'BEGIN {counter=0;} /BEGIN CERT/{counter++} { print > "proxy-cert-" counter ".pem"}' < certs.pem
 ----
 +
-The file `/tmp/pod-cert-chain.pem` contains the workload certificate and the CA certificate propagated to the pod.
-+
-. Verify the root certificate is the same as the one specified by the Operator. Replace `<path>` with the path to your certificates.
+You should have three files in your working dir: `proxy-cert-1.pem`, `proxy-cert-2.pem` and `proxy-cert-3.pem`.
+
+. Verify that the root certificate is the same as the one specified by the administrator. Replace `<path>` with the path to your certificates.
 +
 [source,terminal]
 ----
 $ openssl x509 -in <path>/root-cert.pem -text -noout > /tmp/root-cert.crt.txt
+$ openssl x509 -in ./proxy-cert-3.pem -text -noout > /tmp/pod-root-cert.crt.txt
+$ diff -s /tmp/root-cert.crt.txt /tmp/pod-root-cert.crt.txt
 ----
+The /tmp/root-cert.crt.txt and /tmp/pod-root-cert.crt.txt flies are identical.
```

Comment:
Let's make it clear that this is a result. I usually format the command line output as code.

You should see the following result:
The /tmp/root-cert.crt.txt and /tmp/pod-root-cert.crt.txt flies are identical.

Reply:
edit made